packages/main/src/cache/persist.ts

```diff
@@ -410,7 +410,7 @@ export function addLabels({
   labels.forEach(label => {
     db.prepare(
       `
-      insert into message_gmail_labels (id, label, message_id)
+      insert into message_gmail_labels (label, message_id)
```

Comment:
You'll notice that you got a test failure after this change:

> UnhandledPromiseRejectionWarning: SqliteError: 3 values for 2 columns

Every value that you `select` is used as an inserted value. You declare that you are going to insert values for `label` and `message_id`, and you select `@accountId`, `@label`, `messages.id`. Sqlite attempts to insert the value `@accountId` for the `label` of the new row, the value `@label` for `message_id`, and then notices that you did not provide a slot for the value `messages.id` to be inserted.